tools/mksyscall.c

```diff
@@ -497,14 +515,16 @@ static void generate_stub(int nfixed, int nparms)
    * value, just return zero (OK).
    */
 
-  if (strcmp(g_parm[RETTYPE_INDEX], "void") == 0)
+  fprintf(stream, ");\n");
+  if (strcmp(g_parm[RETTYPE_INDEX], "noreturn") == 0)
     {
-      fprintf(stream, ");\n  return 0;\n}\n");
+      fprintf(stream, "  PANIC();\n");
```

Comment:
Don't need because:
1. We should trust the implementation of noreturn never return
2. It waste the flash space for each PANIC

Reply:
True! It is enough not to generate the "return 0" there. I will fix this asap...

Reply:
Ok, now removed that change and things are still working